lib/src/argument_list_visitor.dart

```diff
@@ -195,14 +193,14 @@ class ArgumentListVisitor {
       // instead of just having this little solo split here. That would try to
       // keep the parameter list with other arguments when possible, and, I
       // think, generally look nicer.
-      if (_functions.first == _allArguments.first) {
+      if (_functions!.first == _allArguments.first) {
```

Comment:
Consider adding a local variable so that these `!`s aren't necessary, e.g.:

```dart
var _functions = this._functions;
if (_functions != null) {
  // ...
  if (_functions.first == _allArguments.first) {
  ...
}
```